### PR TITLE
Adding smart-room-lite skill

### PR DESCRIPTION
**Pull request by singhsaumya**
## Purpose
Adding dialog model json file for official documentation hosted on docs.microsoft.com

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
N/A

* Test the code
N/A

## What to Check
The json file should be successfully imported in the Speech Studio > Custom Commands

## Other Information
N/A

### CODE REVIEW

File: custom-commands/smart-room-lite/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Purpose
+This application is built with the goal of being used as a quick-start for documentation hosted on docs.microsoft.com
+Alternatively, it can also be used by the users to verify their changes, in case they have difficult following through the documentation.
+
+Here, we have tried to use most of the features provide by the Custom Commands platform to build a simple application.
+
+## Links
+
+For usage, refer to official [documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/custom-commands)
```
Comment (chschrae) [inline custom-commands/smart-room-lite/README.md:11]: This link does not provide any usage information. What scenarios does this app provide? Example sentences?

File: custom-commands/smart-room-lite/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Smart Room Lite - A Custom Commands application
+
+## Purpose
+This application is built with the goal of being used as a quick-start for documentation hosted on docs.microsoft.com
+Alternatively, it can also be used by the users to verify their changes, in case they have difficult following through the documentation.
+
+Here, we have tried to use most of the features provide by the Custom Commands platform to build a simple application.
```
Comment (chschrae) [inline custom-commands/smart-room-lite/README.md:7]: "provide" should be "provided"

File: custom-commands/smart-room-lite/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Smart Room Lite - A Custom Commands application
+
+## Purpose
+This application is built with the goal of being used as a quick-start for documentation hosted on docs.microsoft.com
```
Comment (chschrae) [inline custom-commands/smart-room-lite/README.md:4]: docs.microsoft.com should be a link to the docs you are referring to

File: custom-commands/smart-room-lite/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Alternatively, it can also be used by the users to verify their changes, in case they have difficult following through the documentation.
```
Comment (chschrae) [inline custom-commands/smart-room-lite/README.md:5]: "difficult" should be "difficulty" or "a difficult time"

File: custom-commands/smart-room-lite/README.md (hunk 1 above)
Comment (dargilco) [reply custom-commands/smart-room-lite/README.md:11]: Yeah, the link shuold take you right to the quick start page which talks about building this demo

File: custom-commands/smart-room-lite/skill/Smart-Room-Lite-Demo.json [hunk 5]
```diff
@@ -0,0 +1,372 @@
+{
```
Comment (dargilco) [inline custom-commands/smart-room-lite/skill/Smart-Room-Lite-Demo.json:1]: Either use camel-case or dash style, but not both, so
SmartRoomLiteDemo.json, or
smart-room-lite-demo.json

File: custom-commands/smart-room-lite/README.md (hunk 1 above)
Comment (singhsaumya) [reply custom-commands/smart-room-lite/README.md:11]: I have changed ref to point to the currently hosted documentation. We will need to revisit this, once we have the new documentation changes in -- cause the link address would change at that point.

File: custom-commands/smart-room-lite/README.md (hunk 3 above)
Comment (singhsaumya) [reply custom-commands/smart-room-lite/README.md]: I have changed ref to point to the currently hosted documentation. We will need to revisit this, once we have the new documentation changes in -- cause the link address would change at that point.

File: custom-commands/smart-room-lite/README.md (hunk 1 above)
Comment (singhsaumya) [reply custom-commands/smart-room-lite/README.md:11]: @chschrae - I have modified the readme file as per your recommendation to include domain and example sentences. Can you please take a look?

File: custom-commands/smart-room-lite/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Smart Room Lite - A Custom Commands application
+
+## Purpose
+This application is built with the goal of being used as a quick-start for official [documentation]https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/custom-commands.
```
Comment (dargilco) [inline custom-commands/smart-room-lite/README.md:4]: you are missing parenthesis around the URL

File: custom-commands/smart-room-lite/README.md (hunk 1 above)
Comment (singhsaumya) [reply custom-commands/smart-room-lite/README.md:28]: Resolving as discussed offline

**Pull request merged**